src/util/getUniqueRandomNumbers.js

```diff
@@ -0,0 +1,18 @@
+function createNumberArray(min, max) {
+  return Array.from({ length: max - min + 1 }, (_, i) => i + min);
+}
+
+function shuffleArray(array) {
+  for (let i = array.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [array[i], array[j]] = [array[j], array[i]];
+  }
+  return array;
+}
+
+export function getUniqueRandomNumbers(numberRange, count) {
+  const { min, max } = numberRange;
```

Comment:
어차피 객체로 받을거라면 모두 객체로 받아도 좋을거 같아요

```suggestion
export function getUniqueRandomNumbers({ min, max, count }) {
```